tests/npm-tsoa-app/package-lock.json

```diff
@@ -0,0 +1,1765 @@
+{
```

Comment:
Do we really need a lockfile for this?

Reply:
It's common practice. Adding it to .gitignore will give the impression that this is what you're supposed to do for your own projects. I'll leave it to you though, we can also add a note somewhere that the package-lock.json file is only git-ignored because this is a test and not a real app and we don't want to blow up the repo.

Reply:
This is a sample though, and we probably won't refresh this often if at all, I would prefer to have it in .gitignore and use recent dependencies in the CI.

Other than the bloat, pinning means we'll stay on some arbitrary set of old stuff in practice (these are not real dependencies for us), and users that picks this up may get breaks against modern versions (or be forced into needlessly old dependencies instead). We make the same tradeoff for the Python tests, and I think it is the right tradeoff.